components/events/EventCard.vue

```diff
@@ -1,10 +1,5 @@
 <template>
-    <component
-        :is="!!to ? 'locale-link' : !!href ? 'ext-link' : 'div'"
-        :class="cardClasses"
-        :to="to"
-        :href="href"
-    >
+    <div :class="cardClasses">
```

Comment:
the cards do not have to be clickable?

Reply:
Yeah, this is confirmed by @anan0202. She decided to make all the cards have a uniform behavior that hovers to increase the focus because some cards are not clickable, misleading people to consider that is a bug.

Reply:
Great!

Reply:
I checked with @anan0202 again this afternoon. And I found that I misunderstood her meaning last time. The correct behavior should be all the cards should be able to hover and clickable except for those events that don't have links or pages, we should leave the cursor default instead of the pointer.

Reply:
Ok got it!